src/common/iconv.c

```diff
@@ -73,11 +73,19 @@ int guac_iconv(guac_iconv_read* reader, const char** input, int in_remaining,
         /* Read character */
         read_start = *input;
         value = reader(input, in_remaining);
+        /* Stop if not enough bytes to read */
+        if (*input - read_start == 0) {
```

Comment:
To keep style consistent across the code, and since this is a single line of code in the `if` statement, there's no reason for the braces.